types/services/methods.go

```diff
@@ -276,6 +276,7 @@ func (s Service) OnlineDaysPercent(days int) float32 {
 func (s *Service) OnlineSince(ago time.Time) float32 {
 	failsList := s.FailuresSince(ago).Count()
 	hitsList := s.HitsSince(ago).Count()
+	totalList := failsList+hitsList
```

Comment:
Do hits include fails, if so then this doesn't work.

Reply:
Hits should only be successful responses